server.js

```diff
@@ -20,8 +20,9 @@ app.post("/login", (req, res) => {
         success: false
       });
     } else {
+      console.log("result isss: ", result);
       let token = jwt.sign(
-        { username: result.name, pin: result.id },
+        { username: result.name, id: result.id, userRole: result.userRole },
         "Charlie is not a bitch",
```

Comment:
Nice and secure!

Reply:
Seriously though, why not make this instead the output of:

```bash
node -e "console.log(require('crypto').randomBytes(256).toString('base64'));"
```

Reply:
Do you mean to run that in the console and set that as a secret?

Reply:
Yes, read more about this here: https://github.com/dwyl/hapi-auth-jwt2#generating-your-secret-key

Reply:
Awesome, thanks! Changed that. Thanks god for those Dwyl tutorials ;)